js/index.js

```diff
@@ -0,0 +1,147 @@
+    const imageLeft = document.createElement('img');
+    imageLeft.src = project.projectImage;
+    imageLeft.alt = project.companyName;
+    imgLeft.appendChild(imageLeft);
+    projectArticle.appendChild(imgLeft);
+    // Add work Card
+    const workCardRight = document.createElement('div');
+    workCardRight.classList.add('work-section-card-right');
+    // Add company Name
+    const companyName = document.createElement('h1');
+    companyName.classList.add('author-name');
+    companyName.innerText = project.projectName;
+    workCardRight.appendChild(companyName);
+    // Add card Info
+    const cardInfo = document.createElement('ul');
+    cardInfo.classList.add('card-info');
+
+    // Add card info Title
+    const cardInfoLiTitle = document.createElement('li');
+    const cardInfoTitle = document.createElement('p');
+    cardInfoTitle.classList.add('card-info-title');
+    cardInfoTitle.innerText = project.companyName;
+    cardInfoLiTitle.appendChild(cardInfoTitle);
+    cardInfo.appendChild(cardInfoLiTitle);
+
+    // Add dot
+    const cardInfoLiDotOne = document.createElement('li');
+    const imageCounterOne = document.createElement('img');
+    imageCounterOne.src = './img/counter.png';
+    imageCounterOne.alt = 'Counter';
+    cardInfoLiDotOne.appendChild(imageCounterOne);
+    cardInfo.appendChild(cardInfoLiDotOne);
+    // Add Author Position
+    const cardInfoLiPosition = document.createElement('li');
+    const projectPosition = document.createElement('p');
+    projectPosition.classList.add('author-skills');
+    projectPosition.innerText = project.projectPosition;
+    cardInfoLiPosition.appendChild(projectPosition);
+    cardInfo.appendChild(cardInfoLiPosition);
+
+    // Add dot
+    const cardInfoLiDotTwo = document.createElement('li');
+    const imageCounterTwo = document.createElement('img');
+    imageCounterTwo.src = './img/counter.png';
+    imageCounterTwo.alt = 'Counter';
+    cardInfoLiDotTwo.appendChild(imageCounterTwo);
+    cardInfo.appendChild(cardInfoLiDotTwo);
+
+    // Add Date
+    const cardInfoLiDate = document.createElement('li');
+    const projectDate = document.createElement('p');
+    projectDate.classList.add('card-date');
+    projectDate.innerText = project.projectDate;
+    cardInfoLiDate.appendChild(projectDate);
+    cardInfo.appendChild(cardInfoLiDate);
+    workCardRight.appendChild(cardInfo);
+
+    // Add Inner Info
+    const cardInnerInfo = document.createElement('p');
+    cardInnerInfo.classList.add('card-inner-info');
+    cardInnerInfo.innerText = project.projectDescription;
+    workCardRight.appendChild(cardInnerInfo);
+
+    // Add Skills
+    const skills = document.createElement('ul');
+    skills.classList.add('skills');
+    project.projectTechnologies.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('card-skills-li');
+      techItem.innerText = tech;
+      skills.appendChild(techItem);
+    });
```

Comment:
- [ ] As requested by the previous reviewer,  The skills bar is not fitting well on the mobile screen (on the width of 320px), it would be very nice if you make them responsive on the screen. I think this has to do with the height of the popup window. 
_See images below for reference._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123099244-ddad6800-d3e6-11eb-9799-f4af38fd26b4.png) | ![image](https://user-images.githubusercontent.com/49621279/123099347-f4ec5580-d3e6-11eb-88b8-e266389416e6.png)